docs/en_US/Compressor/Overview.md

```diff
@@ -0,0 +1,185 @@
+# Compressor
+NNI provides an easy-to-use toolkit to help user design and use compression algorithms. It supports Tensorflow and PyTorch with unified interface. For users to compress their models, they only need to add several lines in their code. There are some popular model compression algorithms built-in in NNI. Users could further use NNI's auto tuning power to find the best compressed model, which is detailed in [Auto Model Compression](./AutoCompression.md). On the other hand, users could easily customize their new compression algorithms using NNI's interface, refer to the tutorial [here](#customize-new-compression-algorithms).
+
+## Supported algorithms
+We have provided two naive compression algorithms and four popular ones for users, including three pruning algorithms and three quantization algorithms:
+
+|Name|Brief Introduction of Algorithm|
+|---|---|
+| [Level Pruner](./Pruner.md#level-pruner) | Pruning the specified ratio on each weight based on absolute values of weights |
+| [AGP Pruner](./Pruner.md#agp-pruner) | To prune, or not to prune: exploring the efficacy of pruning for model compression. [Reference Paper](https://arxiv.org/abs/1710.01878)|
```

Comment:
can you explain the AGP pruner in short sentence?

Reply:
I dunno how the algorithms works... Maybe @tanglang96 can help?

Reply:
AGP means Automated Gradual Pruning, prune with gradually increasing sparsity while training (given initial sparsity and final sparsity) until the final sparsity reached,  the smallest magnitude weights are masked with zeros.

Reply:
@liuzhe-lz better to add this sentence

Reply:
Seems too long. I added "automated gradual pruning".